lte/cloud/go/services/subscriberdb/obsidian/models/swagger.v1.yml

```diff
@@ -59,6 +61,11 @@ paths:
           type: string
           description: Filter to subscribers assigned the passed IP address
           required: false
+        - in: query
+          name: verbose
+          type: boolean
+          description: Show full subscribers if true (default) or just show IDs
+          required: false
```

Comment:
(Should be able to) nix this section, since we handle it in the below "endpoint"